libraries/ui-library/src/components/six-sidebar-item-group/six-sidebar-item-group.scss

```diff
@@ -28,3 +28,8 @@
 .six-sidebar-details__header-icon {
   margin-right: 1em;
 }
+
+a {
```

Comment:
Active and visited styles look as follows if rendered as link. Should be overridden to look the same as if rendered as `div`.

*Visited*
![image](https://github.com/six-group/six-webcomponents/assets/1475511/83f0955e-9089-4bab-82f2-5a24d79d690f)

*Active*
![image](https://github.com/six-group/six-webcomponents/assets/1475511/03beb991-a277-44d9-820c-4b5e06d3d523)